### DISCUSSION TITLE
How can I determine if I'm inside a distrobox?  (re: adding label to shell prompt)

### DISCUSSION THREAD

**Discussion by sofakng** [89luca89/distrobox#1100] [Q&A] [answered]
Is there an environment variable (or something else) to determine if I'm inside a distrobox?

For example, I'm using ZSH as my shell but I'd like to add something to my shell prompt so I know I'm inside the distrobox and not on the host.

**Comment by juhp** [comment]
I personally check for `$container` in ~/.bashrc, but that is not unique to distrobox - probably even distro specific.

**Reply by juhp** [reply]
(I do appreciate though that distrobox doesn't overwrite PS1 at least.)

**Comment by 89luca89** [comment, marked as answer]
You can use `$CONTAINER_ID` and `$container` :+1:
